src/simulator/RadioModule.tsx

```diff
@@ -47,43 +47,45 @@ const RadioModule = ({ icon, enabled, minimised }: RadioModuleProps) => {
       ref.current.scrollTop = ref.current.scrollHeight;
     }
   }, [scrollToBottom, items]);
+  if (minimised) {
+    return (
+      <HStack spacing={3}>
+        {icon}
+        <MessageComposer
+          enabled={enabled}
+          onSend={handleSend}
+          minimised={minimised}
+        />
+      </HStack>
+    );
+  }
   return (
-    <HStack pb={minimised ? 0 : 2} pt={0} spacing={3}>
-      {minimised ? (
-        <>
-          {icon}
-          <MessageComposer
-            enabled={enabled}
-            onSend={handleSend}
-            minimised={minimised}
-          />
-        </>
-      ) : (
-        <Stack spacing={3} bg="white" borderRadius="md" p={1}>
-          <Stack spacing={1} p={1}>
-            <VStack
-              p={1}
-              width="100%"
-              h="2xs"
-              onScroll={handleScroll}
-              overflowY="auto"
-              scrollBehavior="smooth"
-              ref={ref}
-            >
-              {items.map((item) => (
-                <ChatItem key={item.id} item={item} />
-              ))}
+    <Stack spacing={3} bg="white" borderRadius="md" p={1}>
+      <Stack spacing={1} p={1}>
+        <VStack
+          p={1}
+          h="2xs"
+          onScroll={handleScroll}
+          overflowY="auto"
+          scrollBehavior="smooth"
+          ref={ref}
+        >
+          {enabled ? (
+            items.map((item) => <ChatItem key={item.id} item={item} />)
+          ) : (
+            <VStack flex="1 1 auto" justifyContent="center">
+              <ChatNotice>The radio is off</ChatNotice>
```

Comment:
I'll do a pass through for i18n in a subsequent PR.

Reply:
Since done as https://github.com/microbit-foundation/python-editor-next/pull/917